pagopa/io-app
# chore(IT Wallet): [SIW-1665] Hide eID card in wallet

> [!WARNING]  
> Depends on https://github.com/pagopa/io-react-native-wallet/pull/145

## Short description
This PR hides the eID card in the wallet screen. It also adds a bottom sheet to display information on the digital identity.

## List of changes proposed in this pull request
- Removed the eID card from the wallet screen
- Added a banner to inform the user the wallet is active
- Added a bottom sheet component to display the eID claims and the issuance date
- Added `expiration` and `issuedAt` properties to `StoredCredential` type, with Redux Persist migration

## How to test
If you have an **existing wallet**, check the following:
- The migration worked, and every credential in the Redux Store now has `issuedAt` and `expiration`
- The eID card is not visible in the wallet screen, only other credentials
- The bottom sheet that appears pressing on "Cos'è?" properly displays the issuance date

Reset the wallet and get all credentials, then check the same points as above.
